data_topic:
  lidar_topic: /livox/lidar
  camera_topic: /pointgrey/image_color/compressed

detection:
  points_number_max: 5000
  points_number_min: 9
  points_number_distance_coeff: 300.0
  mode: "lidar_first"
  boat_pixel_height: 80

filter:
  filter_pt_num: 1
  filter_std: 10.0
  filter_distance_max: 600.0
  filter_distance_min: 100.0

cluster:
  cluster_tolerance: 2.0
  cluster_size_min: 5
  cluster_size_max: 5000

cam:
  cam_net_type: "YOLOV4_TINY"
  cam_file_model_cfg: "/asset/yolov4-tiny-usv.cfg"
  cam_file_model_weights: "/asset/yolov4-tiny-usv_best.weights"
  cam_inference_precison: "FP32"
  cam_n_max_batch: 1
  cam_prob_threshold: 0.5
  cam_min_width: 0
  cam_max_width: 1440
  cam_min_height: 0
  cam_max_height: 1080

extrinsic_parameter:
  translation: [0.0, 0.0, 0.0]
  rotation: [0.0, 0.0, 0.0]

platform:
  scan_cycle_time: 20.0
  scan_range: 80.0
  track_timeout: 10.0
